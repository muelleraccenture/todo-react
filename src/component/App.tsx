import React from 'react';
import './App.css';
import {ItemListContainer} from "./container/ItemListContainer";
import {Provider} from "react-redux";
import {initializeStore} from "../store/store";

export const App = () =>
    <Provider store={initializeStore()}>
      <div className="App">
        <header className="App-header">
          <p>
            TODO
          </p>
        </header>
        <ItemListContainer></ItemListContainer>
      </div>
    </Provider>

export default App;
