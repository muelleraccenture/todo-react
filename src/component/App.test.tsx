import React from 'react';
import App from './App';
import renderer from "react-test-renderer"


describe(App.name, () => {

  it("displays the app", () => {
    const tree = renderer.create(<App/>).toJSON()
    expect(tree).toMatchSnapshot()
  })

})
