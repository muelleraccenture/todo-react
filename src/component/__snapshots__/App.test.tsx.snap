// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App displays the app 1`] = `
<div
  className="App"
>
  <header
    className="App-header"
  >
    <p>
      TODO
    </p>
  </header>
  <div>
    <ul>
      <li
        style={
          Object {
            "backgroundColor": "#B4FCF7",
            "color": "pink",
            "listStyleType": "none",
            "marginTop": "10px",
          }
        }
      >
        <p>
          join experimental noise band
        </p>
        <p />
      </li>
      <li
        style={
          Object {
            "backgroundColor": "#B4FCF7",
            "color": "pink",
            "listStyleType": "none",
            "marginTop": "10px",
          }
        }
      >
        <p>
          become bilingual
        </p>
        <p />
      </li>
    </ul>
  </div>
</div>
`;
